extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-fluid.js

```diff
@@ -84,6 +84,7 @@ describes.realWin('DoubleClick Fast Fetch Fluid', realWinConfig, env => {
     multiSizeImpl.getLayout = getLayout;
     multiSizeImpl.isLayoutSupported('fluid');
     impl.experimentalNonAmpCreativeRenderMethod_ = 'safeframe';
+    multiSizeImpl.experimentalNonAmpCreativeRenderMethod_ = 'safeframe';
```

Comment:
Nit: I'd move line 86 to be immediately after line 83, to group the impls together and the multiSizeImpls together

Reply:
Done.